src/notebooks/serviceRegistry.node.ts

```diff
@@ -28,7 +28,7 @@ import { Identifiers } from '../platform/common/constants';
 import { JupyterDebugService } from './debugger/jupyterDebugService.node';
 import { NotebookIPyWidgetCoordinator } from './controllers/notebookIPyWidgetCoordinator';
 import { RemoteKernelConnectionHandler } from './controllers/remoteKernelConnectionHandler';
-import { JupyterServerSelectorCommand } from './serverSelector';
```

Comment:
I renamed this as I didn't like how we had two files called 'serverSelector.ts' this file has the command, so name it with just the command.